jdl/converters/types.ts

```diff
@@ -88,6 +92,6 @@ export type PostProcessedJSONRootObject = {
   [GENERATOR_JHIPSTER]: PostProcessedJSONGeneratorJhipsterContent;
 };
 
```

Comment:
Is this file types only? If so rename to `types.d.ts`

Otherwise add a type guard operator
```suggestion

export const isJHipsterYoRcContent = (content: any): content is JHipsterYoRcContent => typeof content?.[GENERATOR_JHIPSTER] === 'object';

```